python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -100,7 +100,7 @@ public void testEntrypoint(
             response = invokeHandler(proxy, payload.getRequest(), payload.getAction(), payload.getCallbackContext());
         } catch (final BaseHandlerException e) {
             response = ProgressEvent.defaultFailureHandler(e, e.getErrorCode());
-        } catch (final AmazonServiceException e) {
+        } catch (final AmazonServiceException | AwsServiceException e) {
```

Comment:
[need an import statement for this one as well](https://travis-ci.com/github/aws-cloudformation/cloudformation-cli-java-plugin/jobs/378628075#L4996)

Reply:
Thanks for the catch